docs/scripts/object_orientated_programming_poc/oop_poc_project.py

```diff
@@ -53,35 +53,51 @@ async def store_project():
             datetime(2023, 12, 7, 13, 0, 0, tzinfo=timezone(timedelta(hours=0))),
             datetime(2023, 12, 7, 13, 0, 0, tzinfo=timezone(timedelta(hours=-7))),
         ],
+        "annotation_i_want_to_delete": "I want to delete this annotation",
     }
 
-    # Creating a project =============================================================
+    # 1) Creating a project ==============================================================
     project = Project(
         name="my_new_project_for_testing",
         annotations=my_annotations,
         description="This is a project with random data.",
+        alias="my_project_alias_" + str(uuid.uuid4()).replace("-", "_"),
     )
 
     project = await project.store()
```

Comment:
I am getting a `Lack of READ permission on the requested entity` error on this line.

Reply:
I gave the DPE team permissions on the project